terraform/azure-pipelines.yml

```diff
@@ -1,15 +1,8 @@
-trigger:
-  # automatically runs on pull requests
```

Comment:
I think we need to say `pr: none` so that this doesn't getting triggered on pull requests

https://learn.microsoft.com/en-us/azure/devops/pipelines/yaml-schema/pr?view=azure-pipelines

Reply:
Ah, nvm - we still want `terraform plan` to run for pull requests